Automattic/wp-calypso
# Fix Trustpilot logo for Jetpack checkout

#### Changes proposed in this Pull Request

* Adding a white background Trustpilot logo for Jetpack checkout for the Checkout seals test introduced in https://github.com/Automattic/wp-calypso/pull/34677

#### Testing instructions



* Verify that the testing instructions described in https://github.com/Automattic/wp-calypso/pull/34677 pass
* Now verify on a Jetpack site. Create a Jetpack site using https://jurassic.ninja, click "Setup Jetpack" and go through the signup flow. Add any Jetpack plan to cart and proceed to the /checkout page. Verify that the issue described in https://github.com/Automattic/wp-calypso/issues/35323 is fixed.

Fixes # https://github.com/Automattic/wp-calypso/issues/35323
